chart/templates/scheduler/scheduler-deployment.yaml

```diff
@@ -48,7 +48,7 @@ spec:
 {{- if $stateful }}
   serviceName: {{ .Release.Name }}-scheduler
 {{- end }}
-  replicas: 1
+  replicas: {{ .Values.scheduler.replicas }}
```

Comment:
Please leave this out of this PR -- it's not supported yet until #10956 is merged (in fact we should probably add all of these helm chart changes add this as a commit to there.)